byteosaurus_hex.py

```diff
@@ -1630,6 +1651,10 @@ def tcp_packet(fuzzy, module, tcp_inputs):
                 src_mac, dst_mac = RandMAC()._fix(), RandMAC()._fix()
                 src_ip, dst_ip = RandIP("172.16.0.0/12")._fix(), RandIP("172.16.0.0/12")._fix()
                 tcp_sport, tcp_dport = RandShort()._fix(), RandShort()
+            elif module == 'MPLS':
```

Comment:
Same here as well, since both sections are same worth doing 

```python
if module == 'VXLAN' or module == 'MPLS'
```

Reply:
Addressed

Reply:
latest commit missed this section.. 

```python
1632   │     if fuzzy == 'y':
1633   │         if module != None:
1634   │             if module == 'VXLAN':
1635   │                 src_mac, dst_mac = RandMAC()._fix(), RandMAC()._fix()
1636   │                 src_ip, dst_ip = RandIP("172.16.0.0/12")._fix(), RandIP("172.16.0.0/12")._fix()
1637   │                 tcp_sport, tcp_dport = RandShort()._fix(), RandShort()
1638   │             elif module == 'MPLS':
1639   │                 src_mac, dst_mac = RandMAC()._fix(), RandMAC()._fix()
1640   │                 src_ip, dst_ip = RandIP("172.16.0.0/12")._fix(), RandIP("172.16.0.0/12")._fix()
1641   │                 tcp_sport, tcp_dport = RandShort()._fix(), RandShort()
```